tools/csv-merger/csv-merger.go

```diff
@@ -439,7 +439,15 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 			templateStrategySpec.Permissions = append(templateStrategySpec.Permissions, permissions...)
 
 			ocsCSV.Spec.CustomResourceDefinitions.Owned = append(ocsCSV.Spec.CustomResourceDefinitions.Owned, csvStruct.Spec.CustomResourceDefinitions.Owned...)
-			ocsCSV.Spec.CustomResourceDefinitions.Required = append(ocsCSV.Spec.CustomResourceDefinitions.Required, csvStruct.Spec.CustomResourceDefinitions.Required...)
+
+			for _, definition := range csvStruct.Spec.CustomResourceDefinitions.Required {
```

Comment:
Is this in place because the NooBaa CSV we're importing isn't updated to reflect this change? If so, shouldn't the change be in NooBaa?